openshift-knative-operator/pkg/common/label.go

```diff
@@ -0,0 +1,20 @@
+package common
+
+import (
+	mf "github.com/manifestival/manifestival"
+	socommon "github.com/openshift-knative/serverless-operator/pkg/common"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// InjectCommonLabel injects the common label into the resources.
+func InjectCommonLabel() mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
```

Comment:
Should not the transformer be applied on namespaces only?

Reply:
Do you have any reason why you think so? I don't have any strong opinion actually so it is alright to apply on namespace, though.

Reply:
My thinking was that might be too much to add a label everywhere a bit verbose that is why I asked the rationale behind it.
But also it can be useful in certain cases eg. query by label etc.

Reply:
Okay, I think it is alright to add the labels into namespaces only so updated the PR.